cpp/src/cider/common/interpreters/AggregationHashTable.h

```diff
@@ -190,6 +198,8 @@ class AggregationHashTable final {
   AggregatedHashTableWithUInt16Key agg_ht_uint16_;
   AggregatedHashTableWithUInt32Key agg_ht_uint32_;
   AggregatedHashTableWithUInt64Key agg_ht_uint64_;
+  AggregatedHashTableWithKeys128 agg_ht_uint128_;

```

Comment:
it's Uint128 hashmap or int128 hashmap?

Reply:
It's UInt128 and UInt256, PR&Jira titles changed.

Reply:
But it can be used for short strings, so it's named `Keys128`.

Reply:
And I think we should not name these classes as  `AggregatedHashTableWithXXXkey`, `AggregatedHashTableForXXXkey` would be better.

Reply:
Emm, do we need to change all names above from `AggregatedHashTableWithXXX` to `AggregatedHashTableForXXX` ?

Reply:
I prefer to do so. you can pull someone else to give more feedback about naming.

Reply:
I also prefer `for` rather than `with`, renamed.